src/main/java/br/com/caelum/vraptor/simplemail/MailerFactory.java

```diff
@@ -35,9 +35,6 @@ public Mailer getInstance() {
 	}
 
 	private Mailer grabInstance() {
-		if (env.getName().equals("development")) {
-			return new MockMailer();
-		}
```

Comment:
I don't think that this is a problem @Turini 
Old behavior: MockMailer was used regardless of the "mailer.implementation" configuration.
New behavior: MockMailer is used if "mailer.implementation" is not configured, otherwise the impl from "mailer.implementation" is used.

The VRaptor3 version behaves exactly the same as the "new behavior" if I read the code correctly.
There was also a VRaptor3 commit (31970efa6ce701f1fb873448616898574d03f2f6) which enabled emails in the development env - exactly what this patch does.

So in **my** humble opinion, no new toggle is required.